manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "YOUR APP NAME",
+  "name": "pwa-gtm-pixel",
   "vendor": "vtex",
   "version": "0.0.1",
-  "title": "CHANGEME",
-  "description": "CHANGEME",
+  "title": "PWA Google Tag Manager Pixel",
+  "description": "VTEX App that integrates PWA events with Google-tag-manager",
   "billingOptions": {
     "termsURL": "",
     "support": {
```

Comment:
You can remove pixel builder, since the folder pixel will be empty

Reply:
The pixel builder isn't necessary for the configuration on `plugins.json` and `interfaces.json`?

Reply:
nope, not related